client/components/data/query-jetpack-connection/README.md

```diff
@@ -28,7 +28,7 @@ export default function MyJetpackConnectionStatus( { jetpackConnection } ) {
 
 <table>
 	<tr><th>Type</th><td>Number</td></tr>
-	<tr><th>Required</th><td>Yes</td></tr>
+	<tr><th>Required</th><td>No</td></tr>
```

Comment:
Let's keep this one "Yes". Although the `siteId` can be `null` and it's not required in the context of `PropTypes`, this query component can't function without it.